scripts/CHANGELOG.md

```diff
@@ -0,0 +1,54 @@
+# Change Log - @fluentui/scripts
```

Comment:
Same, do we want this for a net-new package?

Reply:
i can get rid of this for now :)